pymedphys/_dicom/rtplan/core.py

```diff
@@ -212,7 +212,7 @@ def get_leaf_jaw_positions_for_type(
         matching_type = [
             item
             for item in sequence
-            if item.RTBeamLimitingDeviceType == rt_beam_limiting_device_type
+            if item.RTBeamLimitingDeviceType.startswith(rt_beam_limiting_device_type)
```

Comment:
I have concerns about using strings that are not a valid defined type for the rt_beam_limiting_device_type for partial matching.
Better to pass in a list of acceptable types to be matched.
You already have something that raises an error if the list of matching_types is != 1.
so something like rt_beam_limiting_device_type_list in item.RTBeamLimitingDeviceType.

Reply:
Yeah, I was hesitant about this. You concern here cements that.

Reply:
issue specified in https://github.com/pymedphys/pymedphys/issues/1139